assets/Styles/main.css

```diff
@@ -0,0 +1,904 @@
+}
+
+#pastEvents {
+  width: 100%;
+  background-color: white;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 1rem 1rem;
+  border-top: 1px solid #cecfd1;
+}
+
+#pastEvents h5 {
+  color: var(--primary-black);
+  text-align: center;
+  font-family: var(--secondary-ff);
+  font-weight: 600;
+  font-size: 18px;
+  padding-top: 1.5rem;
+}
+
+#pastEvents p {
+  width: 80%;
+  font-family: var(--secondary-ff);
+  font-size: 12px;
+  color: var(--primary-black);
+  text-align: center;
+  padding-bottom: 1rem;
+}
+
+#pastEvents img {
+  margin: 0.5rem 0;
+}
+
+#pastEvents img:nth-child(even) {
+  margin-bottom: 2rem;
+}
+
+.past-event {
+  align-self: center;
+}
+
+.deskShow {
+  display: none;
+}
+
+#partnersAbt {
+  width: 100%;
+  background-color: var(--primary-black);
+  padding: 2rem 1rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+#partnersAbt h2 {
+  font-size: 18px;
+  color: var(--primary-white);
+  text-align: center;
+}
+
+@media only screen and (min-width: 768px) {
+  body {
+    width: 100vw;
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  #menuMobile {
+    display: none;
+  }
+
+  nav {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    width: 100vw;
+    height: 100px;
+    padding: 3rem 4rem 2rem 4rem;
+    box-shadow: 0 -1px 6px 2px rgba(0, 0, 0, 0.25);
+  }
+
+  nav img {
+    width: 130px;
+    height: auto;
+  }
+
+  nav a {
+    text-decoration: none;
+    color: var(--primary-black);
+    margin: 0 1rem;
+  }
+
+  nav .logoNav {
+    width: 35%;
+  }
+
+  nav button {
+    width: 15%;
+    padding: 0.5rem 1rem;
+    border: 4px solid var(--primary-red);
+    background-color: var(--background);
+    margin: 0 1rem 1rem 1rem;
+    height: auto;
+    cursor: pointer;
+    transition: all 0.5s ease-in-out;
+  }
+
+  nav button:hover {
+    background-color: var(--primary-red);
+    border: 2px solid white;
+    color: white;
+  }
+
+  #socialBar {
+    width: 100%;
+    height: 60px;
+    background: black;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+  }
+
+  #socialBar .social-icons {
+    width: 100%;
+    padding-right: 3rem;
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: flex-end;
+  }
+
+  #partContainer img {
+    width: 20%;
+  }
+
+  #aboutLogo img {
+    width: 40%;
+    height: auto;
+    padding: 3rem 4rem;
+    margin-top: 2rem;
+  }
+
+  #pastEvents img {
+    margin: 0 1rem;
+  }
+
+  #socialBar .social-icons img {
+    margin: 0 1rem;
+    width: auto;
+    height: 25%;
+    cursor: pointer;
+  }
+
+  #socialBar .social-icons a {
+    margin: 0 1rem;
+    text-decoration: none;
+    color: var(--primary-white);
+  }
+
+  #heroSection {
+    width: 100%;
+    align-items: center;
+    padding-bottom: 7rem;
+    background-image:
```

Comment:
- [ ] **These background images are not loaded in the live version of your website. Please fix it.**

The images are not loaded probably due to the bad path, please try to use a relative path. 

![image](https://user-images.githubusercontent.com/59168430/125126917-70554300-e119-11eb-9a71-19a6445d9aab.png)

Reply:
Fixed :heavy_check_mark: